src/ResourceManager/RedisCache/ChangeLog.md

```diff
@@ -0,0 +1,21 @@
+﻿<!--
```

Comment:
@SiddharthChatrolaMs this file isn't necessary -- the existing change log for RedisCache can be found [here](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/RedisCache/Commands.RedisCache/ChangeLog.md).

Reply:
This is fixed